cypress/integration/mouse.spec.ts

```diff
@@ -36,6 +36,32 @@ describe("cy.realMouseDown and cy.realMouseUp", () => {
         .realMouseDown({ position: "bottomRight" })
         .realMouseUp({ position: "bottomRight" });
     });
+
+    describe("options.button", () => {
+      it("should allow to press down mouse using middle button", (done) => {
+        cy.get(".action-btn")
+          .then(($button) => {
+            $button.get(0).addEventListener("mousedown", (ev) => {
```

Comment:
Sadly the `pointerdown` event is not being called correctly. I've reported this issue here: https://bugs.chromium.org/p/chromium/issues/detail?id=1276003